### ISSUE TITLE
Importing custom models

### ISSUE THREAD

**Issue by 0skar2** [futo-org/voice-input#95]
Are you planning to implement a way to import your own models, like in your keyboard app? This is really nice app, but the Polish model is really, really inaccurate, and i'd like to import a model specifically trained in Polish.

I couldnt find an option like that anywhere, and if im blind, im sorry.

**Comment by metrixsam** [comment]
I agree. I believe that the voice input app should work too even, just like the keyboard app does and relies on importing local resources files. Though without eliminating automatic language detection just like what happened with the keyboard app.
